src/Akeneo/UserManagement/Bundle/Resources/public/js/form/user-interface-helper-view.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {ThemeProvider} from 'styled-components';
+import {pimTheme} from 'akeneo-design-system';
+import {ReactView} from '@akeneo-pim-community/legacy-bridge/src/bridge/react';
+import {DependenciesProvider} from '@akeneo-pim-community/legacy-bridge';
+import {UserInterfaceHelper} from "./user-interface-helper";
+import {QueryClient, QueryClientProvider} from "react-query";
+
+class UserInterfaceHelperView extends ReactView {
```

Comment:
I would suggest a more explicit naming in place of "User interface helper".
Something like "Unsupported ui locale warning".

```suggestion
class UnsupportedUiLocaleWarningView extends ReactView {
```